### DISCUSSION TITLE
Wireguard Monitoring

### DISCUSSION THREAD

**Discussion by ArtemGoblin** [wg-easy/wg-easy#628] [Q&A] [answered]
Hello. 
Thank you, everything works as it should.
There is a question about how to set up logging to collect analysis of Wiregurad user traffic. I looked at third-party solutions, but did not find a suitable option

**Comment by Justieku** [comment]
+

**Comment by verum-haeresis** [comment]
!

**Reply by Justieku** [reply]
Ok, I got you

**Comment by BETEP-ok** [comment]
Double that. A feature I am terribly missing… some basic stats:
Total traffic consumed/per client/per time frame.
Current total up/down speed (and per client)
Average speed per time frame.

even that could be already enough.!

**Comment by centralhardware** [comment, marked as answer]
https://github.com/MindFlavor/prometheus_wireguard_exporter i use this prometheus exporter
